### DISCUSSION TITLE
Make Battery Upgrade able to charge items in curio and inventory slots

### DISCUSSION THREAD

**Discussion by Joly0** [P3pp3rF1y/SophisticatedBackpacks#563] [Ideas]
It would be a great addition to the battery upgrade if it were able to recharge items in the inventory and curio slots.
Additionaly some sort of configurability would be nice so one could define which items get recharged in which slows (all inventory, curio, armor, held item, hotbar). Additionaly an option for quick charge or instant charge would be great if put into the charge slot (or discharge slot for that matter)

I think this would make a great addition to the battery upgrade it would make it more useful for longer adventures as one would have to always put their items in the charge slot in the backpack and wait for it to recharge.

**Comment by isaaclepes** [comment]
I just came here to request the same thing. I use the wireless crafting grid from Refined Storage and it can charge from a wireless charger while in a curio slot, but not from the backpack battery. I either have to move the item back into the main inventory, or return to the radius of a wireless charger.
